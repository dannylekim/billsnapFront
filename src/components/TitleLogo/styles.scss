.title {
  max-width: 500px;
}

.title__row {
  display: flex;
  justify-content: space-evenly;

  &.title__text {

  }

  .title__title {

  }

  .title__logo {

  }
}
