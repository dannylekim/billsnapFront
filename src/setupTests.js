// setup file
/** 
 * DON'T TOUCH THIS FILE.
 * IT'S REQUIRED TO TEST COMPONENTS
*/
import Enzyme, { shallow, render, mount } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer  from 'react-test-renderer';

configure({ adapter: new Adapter() });

const matches = children => expect(
    renderer.create(children).toJSON()
  ).toMatchSnapshot();
  
global.shallow = shallow;
global.render = render;
global.mount = mount;
global.matches = matches;
global.localStorage = {
  setItem: jest.fn()
};