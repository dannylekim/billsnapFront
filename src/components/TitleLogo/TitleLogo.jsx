import React from "react";
import "./styles.scss";

export const TitleLogo = () => {
  return (
    <div className="title__row">
      <div className="title__title">
        <h1>Billsnap</h1>
      </div>
      <div className="title__logo">
        <img
          src="http://via.placeholder.com/150x100.png?text=150x100+Logo"
          alt="Logo"
        />
      </div>
    </div>
  );
};

export default () => {
  return (
    <div className="title">
      <TitleLogo />
      <div className="title__row title__text">
        <p>
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque
          dignissimos eos error quo, repellendus repudiandae sunt. Amet
          asperiores atque commodi, deleniti dicta expedita ipsa modi nemo, quos
          rem voluptate voluptates.
        </p>
      </div>
    </div>
  );
};
