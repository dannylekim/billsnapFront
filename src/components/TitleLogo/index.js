import TitleLogo from "./TitleLogo";
export * from './TitleLogo';
export default TitleLogo;