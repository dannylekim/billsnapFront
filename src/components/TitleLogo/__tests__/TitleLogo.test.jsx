import React from "react";
import ReactDOM from "react-dom";
import { TitleLogo } from "../TitleLogo.jsx";
import {matches} from "../../../setupTests";

describe("TitleLogo", () => {
  describe("render", () => {
    it("renders without crashing", () => {
      const div = document.createElement("div");
      ReactDOM.render(<TitleLogo />, div);
    });
  });
});

describe('TitleLogo', () => {

  describe("render", () => {
    describe("snapshots ðŸ“¸", () => {
      it("TitleLogo should match snap shot", () => {
        matches(<TitleLogo />);
      });
    });

    describe("components", () => {});
  });

  describe("functions", () => {});
});
