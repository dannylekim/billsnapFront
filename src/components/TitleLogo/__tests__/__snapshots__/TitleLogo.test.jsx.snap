// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleLogo render snapshots ðŸ“¸ TitleLogo should match snap shot 1`] = `
<div
  className="title__row"
>
  <div
    className="title__title"
  >
    <h1>
      Billsnap
    </h1>
  </div>
  <div
    className="title__logo"
  >
    <img
      alt="Logo"
      src="https://via.placeholder.com/150x100.png?text=150x100+Logo"
    />
  </div>
</div>
`;
